src/app/typescript/v5/migrate/page.mdx

```diff
@@ -1,13 +1,82 @@
 # Migration from SDK v4
 
-### High-level changes
+## Why you should migrate to SDK v5
+
+#### 1. Better performance, happier clients
+The new SDK is built with performance in mind and proper tree-shaking. Therefore, the minimum bundle size of your application is greatly reduced.
+
+Below is the comparison of 2 similar applications, using `ConnectWallet` (v4) & `ConnectButton` (v5), which are identical in term of functionality.
+
+|                     | SDK v4 | SDK v5
+| ------------------- | ------ | -------
+| Minimum bundle size | <span style={{color: 'red'}}>766kb</span>  | <span style={{color: 'green'}}>104kb</span>
+| Dependencies        | `"@thirdweb-dev/react": "^4.9.4"`<br/>`"@thirdweb-dev/sdk": "^4.0.99"`<br/>`"ethers": "^5.7.2"`| `"thirdweb": "^5.42.0"`
+
+_(Built with Next.js 14.2.5)_
+
+#### 2. More wallets supported
+The SDK v4 only supports a handful of web3 wallets and the more wallets you want to include in your app, the heavier it becomes.
+
+SDK v5 supports over 300 wallets and this number is increasing! You can [interact with wallets based on their unique IDs](/typescript/v5/wallets#example-connect-a-wallet-and-access-an-account-to-send-a-transaction).
+Hence, adding more wallets to your app has little to no effect to the final bundle.
+
+#### 3. Flexibility with React hooks
```

Comment:
would love to see a specific TS and React migration guides after this, splitting the 2 for very guided flows